core/src/main/java/io/questdb/griffin/engine/functions/rnd/RndStringRndListFunctionFactory.java

```diff
@@ -94,12 +85,18 @@ public void close() {
 
         @Override
         public CharSequence getStr(Record rec) {
+            if (nullRate > 0 && (rnd.nextInt() % nullRate) == 1) {
```

Comment:
minor: why not `nextPositiveInt()`? Mod over negative will be negative and never 1, meaning effective `nullRate` will be halved.

Reply:
> minor: why not `nextPositiveInt()`? Mod over negative will be negative and never 1, meaning effective `nullRate` will be halved.

found this too, fixed it. thank you.